### ISSUE TITLE
Chapter 11 - 'CallbackHandler' object has no attribute 'tokenizer' error for SetFit trainer

### ISSUE THREAD

**Issue by weigo2002** [HandsOnLLM/Hands-On-Large-Language-Models#19]
trainer = SetFitTrainer(
    model=model,
    args=args,
    train_dataset=sampled_train_data,
    eval_dataset=test_data,
    metric="f1"
)

Got 'CallbackHandler' object has no attribute 'tokenizer' error  when initialize SetFitTrainer.

**Comment by MaartenGr** [comment]
Thank you for sharing! It seems that an update of `transformers` broke down the functionality of `SetFit`. To resolve the issue, I would advise checking out the thread [here](https://github.com/huggingface/setfit/issues/564). Essentially, it would mean temporarily using an older version of Transformers until this gets fixed.
